backend/server.js

```diff
@@ -25,6 +26,9 @@ app.use(express.static('public'))
 app.use(express.urlencoded({ extended: true }))
 app.use(express.json())
 
+// CORS enabling middleware
+app.use(cors());
```

Comment:
Smart! This makes it so you don't have to use the developer menu settings to enable calls to the backend, right?

Reply:
Yes! This node.js CORS package automatically allows any domain access the backend, very handy!